### PR TITLE
DTGB-741: Added fixes for JS binding files.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: gent_base.theme [hunk 1]
```diff
@@ -122,7 +122,7 @@ function gent_base_preprocess_page(&$variables) {
 function gent_base_preprocess_page_hero_image(&$variables) {
   $route = \Drupal::request()->get('_route');
   if ($route !== 'entity.node.canonical') {
-    $variables['attributes']['class'][] = 'overview-page';
+    $variables['attributes']['class'][] = ['overview-page', 'overview-layout'];
```
Comment (hstandaert) [inline gent_base.theme]: what does this do? 🤔

Comment (meirege) [reply gent_base.theme]: You need this class if you want the .intro class to work...

Comment (hstandaert) [reply gent_base.theme]: I don't think the `overview-layout` class is intended for this... We should either move the `intro` class to a global level in styleguide; or add some custom styling in the desired project

File: gent_base.theme [hunk 2]
```diff
@@ -122,7 +122,7 @@ function gent_base_preprocess_page(&$variables) {
 function gent_base_preprocess_page_hero_image(&$variables) {
   $route = \Drupal::request()->get('_route');
   if ($route !== 'entity.node.canonical') {
-    $variables['attributes']['class'][] = 'overview-page';
+    $variables['attributes']['class'][] = ['overview-page'];
```
Comment (hstandaert) [inline gent_base.theme]:
```suggestion
    $variables['attributes']['class'][] = 'overview-page';
```

**Review by hstandaert — APPROVED** [review]

**Pull request merged**